api/app/serializers/v1/stylesheet_serializer.rb

```diff
@@ -3,15 +3,23 @@ class StylesheetSerializer < ManifoldSerializer
 
     include ::V1::Concerns::ManifoldSerializer
 
-    typed_attribute :name, NilClass
-    typed_attribute :source_identifier, NilClass
-    typed_attribute :styles, NilClass
-    typed_attribute :ingested, NilClass
-    typed_attribute :position, NilClass
-    typed_attribute :created_at, NilClass
+    typed_attribute :name, Types::String
+    typed_attribute :source_identifier, Types::String.optional.meta(read_only: true)
+    typed_attribute :styles, Types::String.optional.meta(
```

Comment:
Please change the description to:

"Validated, scoped, and transformed styles that will be applied in the reader."

Reply:
Done